Bottom: 75153e86dee441c11639cc0d142e7aa67c2eeb66
Top:    f0f8fa3629cabc6f95ecf42c902897ec4f755243
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-02-16 16:54:13 +0800

Refresh of virtio-net-optimize-small

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 98a7e93..aa7dc5d 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -803,7 +803,7 @@ static int add_recvbuf_small(struct virtnet_info *vi, struct receive_queue *rq,
 	struct page_frag *alloc_frag = &rq->alloc_frag;
 	char *buf;
 	unsigned int xdp_headroom = virtnet_get_headroom(vi);
-	int len = hdr_len + NET_IP_ALIGN + GOOD_PACKET_LEN + xdp_headeroom;
+	int len = hdr_len + NET_IP_ALIGN + GOOD_PACKET_LEN + xdp_headroom;
 	int err;
 
 	len = SKB_DATA_ALIGN(len) +
@@ -813,7 +813,7 @@ static int add_recvbuf_small(struct virtnet_info *vi, struct receive_queue *rq,
 
 	buf = (char *)page_address(alloc_frag->page) + alloc_frag->offset;
 	sg_init_table(rq->sg, 1);
-	sg_set_buf(rq->sg + 1, buf + NET_IP_ALIGN + xdp_headeroom,
+	sg_set_buf(rq->sg + 1, buf + NET_IP_ALIGN + xdp_headroom,
 		   GOOD_PACKET_LEN);
 
 	err = virtqueue_add_inbuf(rq->vq, rq->sg, 1, buf, gfp);
